shrine_with_cart/mdc_100_series/lib/short_bottom_sheet.dart

```diff
@@ -0,0 +1,578 @@
+import 'dart:async';
+
+import 'package:flutter/material.dart';
+import 'package:flutter/scheduler.dart' show timeDilation;
+import 'package:flutter/services.dart';
+
+import 'package:meta/meta.dart';
+import 'package:scoped_model/scoped_model.dart';
+
+import 'colors.dart';
+import 'model/app_state_model.dart';
+import 'model/product.dart';
+import 'shopping_cart.dart';
+
+class ShortBottomSheet extends StatefulWidget {
+  final AnimationController hideController;
+  const ShortBottomSheet({Key key, this.hideController}) : super(key: key);
+
+  @override
+  _ShortBottomSheetState createState() => _ShortBottomSheetState();
+
+  static _ShortBottomSheetState of(BuildContext context, {bool nullOk: false}) {
```

Comment:
Ok! I modeled this after `MediaQueryData.of`, which does have `nullOk` as a parameter, so that seemed to be the style, but I can see how it would be confusing.

Reply:
I'd change it. You can always change it back if anyone says it's tradition.

Reply:
Done!